### ISSUE TITLE
Config Volume Control

### ISSUE THREAD

**Issue by TheTerribleT** [Tau-ri-Dev/JSGMod-1.12.2#197]
I've noticed something with previous and even the current versions where the volume control in the config doesn't affect the ring rotation sound of any gate.
I have tried several different versions including the most recent and still experience this. It is kind of weird that the ring is overpoweringly loud and then the chevron locks are very quiet when the volume is like 0.3 in the config.
I am unsure as to what to do about that.

**Comment by MineDragonCZ** [comment]
I can't reproduce this.

**Comment by TheTerribleT** [comment]
Here is a video I made showing how it is not affecting the ring rotation sounds of any of the gates. It isn't so much I want to turn off the sound, more as I cannot turn them DOWN and it is overpowering every other sound.
https://youtu.be/cNIai8fwACc

**Comment by MineDragonCZ** [comment]
> Here is a video I made showing how it is not affecting the ring rotation sounds of any of the gates. It isn't so much I want to turn off the sound, more as I cannot turn them DOWN and it is overpowering every other sound. https://youtu.be/cNIai8fwACc

Yeah, I know what's the problem... You must restart the game after changing the config. Its because the sound is cashed in map with the original volume.

**Comment by TheTerribleT** [comment]
That does make sense. After restarting the game with the adjusted volume it is quieter in all aspects.
